docker/entrypoint.sh

```diff
@@ -1,11 +0,0 @@
-#!/bin/sh
-set -e
-
-if [ -z "$CREDENTIALS" ]; then
-    echo "Must defined CREDENTIALS env var in form of client_name:key"
-    exit 1
-fi;
-
-echo $CREDENTIALS > etc/credentials
-
-dmqnode -c etc/config.ini
```

Comment:
You could just use `exec dmqnode` on this line, to keep the script but have dmqnode receive signals.

Reply:
Hmm, I should have mentioned that the actual reason for removing the script is, that there is just no need for it any longer :). For "real world" uses the etc directory, containing a credentials file, is mounted into the container and this image as it comes provides test credentials.